Assets/Scripts/Enemy.cs

```diff
@@ -11,12 +11,19 @@ public class Enemy : MonoBehaviour
     // Start is called before the first frame update
     public float max_speed = 4;
     public float acceleration_rate = 1;
+
+    // Variables for health and damage management
+    public int max_health = 100;
+    public int curr_health;
```

Comment:
I left max_health and curr_health public for a couple reasons.
1. We can tweak the enemy prefab's max_health and instances' curr_health in the inspector, which is convenient, and I don't see any inherent reason to make it private and use [SerializeField] on it instead (after all, if that were the standard, you'd surely have done that with min_momentum, move_strength, max_speed, and decay_rate, which are used directly nowhere except the Enemy script).
2. In the future, depending on what we decide to do, we **may** benefit from giving other classes access to these variables, such as if we decide to display an enemy health bar. It wouldn't be inherently necessary by my estimation, as if made private, they could be passed to an enemy health bar as method parameters since, unlike the player health bar, there are no foreseeable reasons why we might need to change the value of the curr_health variable within an enemy health bar class.

I don't initialize curr_health as max_health at declaration because I can't, for the same reason we can't initialize _sprite as GetComponent<SpriteRenderer>() on initialization: field initializers can't reference non-static fields or methods. We could make max_health static, but that also makes the assumption that we wouldn't want to change the max_health at any point during runtime for specific instances of Enemy, which actually goes against previous verbal discussions about possible future iterations on Enemy, namely changing the health relative to how many are clustered around the player.